zim/datetimetz.py

```diff
@@ -230,7 +230,7 @@ def replacefunc(matchobj):
 
 def strftime(format, date):
 	# TODO: deprecate this function
-	return date.strftime(format)
+	return date.strftime(format.encode('unicode-escape').decode()).encode().decode('unicode-escape')
```

Comment:
Could you add an inline comment for this? The tests pass on my Linux system without this change, so given the above TODO about deprecation someone further down the line might re-break this if the purpose of this encode-decode chain isn't explained. The current commit message, "avoid strftime failed when accepted unicode string under Windows", would imo suffice, with some more context (source of solution: the internet vs. manual testing?) in the commit message.